src/bin/utils/options.ts

```diff
@@ -118,6 +125,21 @@ const rootKeyPairBits = new commander.Option(
   'Bit size of root key pair',
 ).argParser(binParsers.parseNumber);
 
+const seedNodes = new commander.Option(
+  '-sn, --seed-nodes [nodeId1@host:port;nodeId2@host:port;...]',
+  'Seed node address mappings',
+)
+  .argParser(binParsers.parseSeedNodes)
+  .env('PK_SEED_NODES');
+
+const network = new commander.Option(
+  '-n --network <network>',
+  'Setting the desired default network.',
+)
+  .argParser(binParsers.parseNetwork)
+  .env('PK_NETWORK')
+  .default('mainnet');
```

Comment:
Instead of a string, use `config.defaults.network.mainnet`. Then the expectation would be that `binParsers.parseNetwork` would actually fetch the right POJO based on the string.